core/templates/pages/classroom-page/classroom-page.component.html

```diff
@@ -7,29 +7,66 @@
   <div class="oppia-classroom-banner">
     <img ng-src="<[$ctrl.bannerImageFileUrl]>" align="center" width="100%" alt="">
   </div>
+  <div class="oppia-classroom-details">
+    <h2 translate="I18N_CLASSROOM_PAGE_COURSE_DETAILS"></h2>
+    <p class="course-details"> <[$ctrl.classroomData.getCourseDetails()]> </p>
+    <h2 translate="I18N_CLASSROOM_PAGE_TOPICS_COVERED"></h2>
+    <p> <[$ctrl.classroomData.getTopicsCovered()]> </p>
+  </div>
   <div class="oppia-topic-summary-tiles">
-    <md-card ng-repeat="topicSummary in $ctrl.topicSummaries" class="oppia-topic-summary-tile protractor-test-topic-summary-tile">
+    <div ng-repeat="topicSummary in $ctrl.classroomData.getTopicSummaries()" class="oppia-topic-summary-tile protractor-test-topic-summary-tile">
       <topic-summary-tile topic-summary="topicSummary">
       </topic-summary-tile>
-    </md-card>
-  </div>
-  <div class="oppia-classroom-search-section">
-    <div class="classroom-page-help-text initial-help-text" translate="I18N_CLASSROOM_PAGE_SEARCH_BAR_HEADING"></div>
-    <div class="classroom-page-help-text" style="font-size: 1em;" translate="I18N_CLASSROOM_PAGE_SEARCH_BAR_SUBHEADING"></div>
-    <div style="margin-left: 15vw;">
-      <search-bar enable-dropup="true"></search-bar>
     </div>
   </div>
 </div>
+<div class="oppia-classroom-search-section">
+  <div class="classroom-page-help-text initial-help-text" translate="I18N_CLASSROOM_PAGE_SEARCH_BAR_HEADING"></div>
+  <div class="classroom-page-help-text" style="font-size: 1em;" translate="I18N_CLASSROOM_PAGE_SEARCH_BAR_SUBHEADING"></div>
+  <div>
+    <search-bar enable-dropup="true"></search-bar>
+  </div>
+</div>
 <style>
+  html {
+    background-color: #ffffff;
+  }
+
+  body {
+    background-color: #ffffff;
+  }
```

Comment:
I had to do this to get the background white like in the mocks. Other quick methods I tried with setting a div container for the whole page added the background over the background-banner. I've checked and moving from this page to other pages does work fine and the white color only stays for this page. 
Is this fine?

Reply:
I'm fine with this.

```suggestion
  html, body {
    background-color: #ffffff;
  }
```
Maybe? Also, please change the HEX colour code to a 3 char one in this case (we recently added this to the [styleguide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css)).

Reply:
Mm, prefer to do this without altering the html/body tags if possible. The way it's done here does seem like too broad a scope and makes the CSS less maintainable.

Reply:
Added a class to the html and body tag in `classroom-page.mainpage.html` file and added the style there.